apps/rush-lib/src/cli/actions/ListAction.ts

```diff
@@ -6,15 +6,41 @@ import { BaseRushAction } from './BaseRushAction';
 import { RushCommandLineParser } from '../RushCommandLineParser';
 import { CommandLineFlagParameter } from '@rushstack/ts-command-line';
 import { RushConfigurationProject } from '../../api/RushConfigurationProject';
+import { VersionPolicyDefinitionName } from '../../api/VersionPolicy';
 import { SelectionParameterSet } from '../SelectionParameterSet';
 
 const cliTable: typeof import('cli-table') = Import.lazy('cli-table', require);
 
+/**
+ * Shape of "rush list --json" output.
+ *
+ * It contains parts (with different names) coming from
+ * {@link ../../api/RushConfigurationProject#IRushConfigurationProjectJson | IRushConfigurationProjectJson}.
```

Comment:
I'm unsure if rush-lib do use API Extractor's `@public`, and links. This didn't. Should it? It took time for me to realize this object was a re-shaped version of another.

Reply:
Because `IJsonEntry` is not exported by the `@rushstack/rush-lib` entry point, it is not part of the public API surface, so it does not need a `@public` tag.  (If it did, then API Extractor would report a warning about this.)